src/models/ctx/update_library.rs

```diff
@@ -78,6 +78,25 @@ pub fn update_library<E: Env + 'static>(
             }))
             .unchanged(),
         },
+        Msg::Action(Action::Ctx(ActionCtx::ToggleLibraryItemNotifications(id))) => {
```

Comment:
- [ ] Makes more sense to send a bool with the action to indicate the Subscribing or Unsubscrbing.

Reply:
Why ? This way we don't have to pass the state that the core already know
It will be a toggle/checkbox

Reply:
It's better to send specific state, UI can use toggle functionality but for backends it's better to set the state explicitly.
This removes the cases where you accidentally have 2 requests/actions and you flip the state back to its original state instead of `true`

It's also consistent with other actions.

Reply:
The core is synchronous so this should not happen